exchange/binance.go

```diff
@@ -516,6 +519,10 @@ func (b *Binance) CandlesSubscription(ctx context.Context, pair, period string)
 	return ccandle, cerr
 }
 
+func (b *Binance) CandlesSubscriptionCombined(_ context.Context, _ map[string]string) (chan model.Candle, chan error) {
+	panic("not implemented")
```

Comment:
We do not have this for Spot market?

Reply:
I suggest implementing it in following PR to not overcomplicate this one